### PR TITLE
Add Tommy, Patrice and all spoiled TDE cards from Sep 12 stream

### PR DESCRIPTION
**Pull request by zzorba**

### CODE REVIEW

File: pack/tde/tde.json [hunk 1]
```diff
@@ -83,5 +178,248 @@
...
+		"traits": "Item. Instrument.",
+		"type_code": "asset"
+	},
+	{
+		"code": "06017",
+		"deck_limit": 1,
+		"enemy_damage": 3,
+		"enemy_evade": 5,
+		"enemy_fight": 5,
+		"enemy_horror": 0,
```
Comment (drawntotheflame) [inline pack/tde/tde.json]: I think you can remove this line if there's no horror...

File: pack/tde/tde.json [hunk 2]
```diff
@@ -83,5 +178,248 @@
...
+		"traits": "Madness.",
+		"type_code": "treachery"
+	},	
+	{
+		"code": "06020",
+		"cost": 0,
+		"deck_limit": 3,
+		"faction_code": "guardian",
+		"myriad": true,
+		"name": "Solumn Vow",
```
Comment (drawntotheflame) [inline pack/tde/tde.json]: Solumn to Solemn, please.

File: pack/tde/tde.json [hunk 3]
```diff
@@ -83,5 +178,248 @@
...
+		"pack_code": "tde",
+		"position": 20,
+		"quantity": 3,
+		"skill_willpower": 2,
+		"text": "Myriad. Fast.\nPlay Solumn Vow under the control of another investigator at your location.\n[fast] If the owner of Solumn Vow is at your location, exhaust Solumn Vow: Move 1 damage or horror from a card you control to a card that investigator controls.",
```
Comment (drawntotheflame) [inline pack/tde/tde.json]: Solumn to Solemn.

**Comment by zzorba** [comment]
All fixed, @Kamalisk I think we can merge this.

**Pull request merged**